Makefile

```diff
@@ -0,0 +1,8 @@
+CURRENT_DIRECTORY := $(shell pwd)
```

Comment:
Are variables CURRENT_DIRECTORY and TESTS_TO_RUN used?

Reply:
removed the whole file. No longer needed as we run the tests in teamcity environment